### PR TITLE
Update moj-forensic-readiness-policy.md

### PR DESCRIPTION
**Pull request by cyberapp101**
Hi Adrian. I hope this has been done correctly.

### CODE REVIEW

**Comment by cyberapp101** [comment]
Hi. Just made a small change in this branch regarding inserting the correct link for MOJ Group Security. I'm not sure if you can see the change as I was unable to make a second pull request after making that correction. Please let me know that I have done it correctly. Thanks

File: content/security/framework/moj-forensic-readiness-policy.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Each policy statement outlines a security requirement and where applicable, a re
 Policy statement text.</td></tr>
 </table>
 
-The policies outlined in this document form the baseline standard. Where exceptions are required, this is captured on a case by case basis in Tier 4, where approval is required from both the business group SIRO and MoJ ITSO.
+The policies outlined in this document form the baseline standard. Where exceptions are required, this is captured on a case by case basis in Tier 4, where approval is required from both the business group SIRO and MoJ CISO.
```
Comment (warmanaMOJ) [inline content/security/framework/moj-forensic-readiness-policy.md]: The change is correct, but I'd suggest clarifying the abbreviations ie "Chief Information Security Officer (CISO)".

**Comment by warmanaMOJ** [comment]
Nice, thank you. If you would consider a minor suggestion about expanding the abbreviations, that would be helpful. Otherwise, looks good.

File: content/security/framework/moj-forensic-readiness-policy.md (hunk 1 above)
Comment (cyberapp101) [reply content/security/framework/moj-forensic-readiness-policy.md]: Done!

**Comment by warmanaMOJ** [comment]
@cyberapp101 LGTM, thank you.

@cybersquirrel Please may I request approval to publish these changes?

File: content/security/framework/moj-forensic-readiness-policy.md [hunk 2]
```diff
@@ -189,7 +189,7 @@ All investigations will either be:
 Forensic investigations are only to be carried out under the following circumstances:
 
 - Risk Management of a system has revealed a particularly sensitive/vulnerable area which needs to be proactively monitored. Any discovered security incidents would then be escalated through the IT security incident management process.
-- A business function has issued a request to gather forensic investigation evidence directly to the MoJ Operational Security Team (OST). Results of such an investigation will be handed back to the requesting business function. Any request will be processed through the appropriate incident management process and escalated to the SSA, CISO or SIRO as required.
+- A business function has issued a request to gather forensic investigation evidence directly to the MoJ Operational Security Team (OST). Results of such an investigation will be handed back to the requesting business function. Any request will be processed through the appropriate incident management process and escalated to the Senior Security Advisor (SSA), CISO or SIRO as required.
```
Comment (cybersquirrel) [inline content/security/framework/moj-forensic-readiness-policy.md]: We are rebranding OST - Rob S can advise on the correct naming for the team.

I think an example would be helpful here. What circumstances would/should a business function request a forensic investigation?

The third sentence here ("Any request will...") doesn't immediately make sense to me - I think this is generically true isn't it? Looking at the other bullet points, I think that's the case.

File: content/security/framework/moj-forensic-readiness-policy.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Each policy statement outlines a security requirement and where applicable, a re
 Policy statement text.</td></tr>
 </table>
 
-The policies outlined in this document form the baseline standard. Where exceptions are required, this is captured on a case by case basis in Tier 4, where approval is required from both the business group SIRO and MoJ CISO.
+The policies outlined in this document form the baseline standard. Where exceptions are required, this is captured on a case by case basis in Tier 4, where approval is required from both the business group Senior Information Risk Owner (SIRO) and MoJ Chief Information Security Officer (CISO).
```
Comment (cybersquirrel) [inline content/security/framework/moj-forensic-readiness-policy.md]: What is 'Tier 4' in this?

File: content/security/framework/moj-forensic-readiness-policy.md (hunk 2 above)
Comment (warmanaMOJ) [reply content/security/framework/moj-forensic-readiness-policy.md]: @cyberapp101 ^^^

File: content/security/framework/moj-forensic-readiness-policy.md (hunk 3 above)
Comment (warmanaMOJ) [reply content/security/framework/moj-forensic-readiness-policy.md]: @cybersquirrel It appears to be a 'carry over' from the original (2013) document containing the (then) current version of the policy. It refers to parts of the MOJ, other than HQ, for example HMCTS or ALBs.

@cyberapp101 As per ^^, may I suggest rephrasing from "... on a case by case basis in Tier 4, where approval ..." to " ... on a case-by-case basis by the corresponding group within the MOJ. In each case, exception approval ..." ?

Comment (cybersquirrel) [reply content/security/framework/moj-forensic-readiness-policy.md]: What is "tier 4"?

**Pull request closed without merging**